.cirrus.yml

```diff
@@ -22,8 +22,11 @@ Linux_task:
 
 
   publish_script: |
-    if test "$CIRRUS_TAG" != ""; then
-      for artifactName in runtime/artifacts/packages/Release/Shipping/*.*; do
+    if [ -z "$CIRRUS_PR" ]; then
+      if test "$CIRRUS_TAG" != ""; then
+        CIRRUS_TAG=LATEST
```

Comment:
Do we want to publish artifacts for every run? Current approach is (on-demand) on tag push, e.g. `git tag MyTag && git push --tags`, which I think is cleaner to manage and not overload the resources.

Reply:
I wanted rolling daily build for testing. I tried publishing to blob but I could not get it working quickly. So I decided to piggyback on your work and use LATEST to rolling builds. 
If you have suggestions, please let me know. Our CI is publishing to Azure blob storage and deletes old files. I don't think we need any real history nor sources.

Reply:
`CIRRUS_TAG` corresponds to a well known `git tag` concept in git communities as well as  well-documented GitHub release strategies: https://help.github.com/en/github/administering-a-repository/managing-releases-in-a-repository. If we chose just one `git-tag` of doing things, that would be cleaner to reason about in any kind of CI environment.

Overwriting the build-in a special variable this way is not the best solution and it may also have side effects (pushing artifacts to same release).

I think whichever automation is triggering the daily build can simply run `git tag <tag-name-with-date-and-version> && git push --tags` commands. If it is a manual process, then it is much easier for you to push the tag or even create a tag from UI (even from mobile device) with two clicks: https://github.com/wfurt/freebsd-bootstrap-cli/releases/new. We can follow same naming convention for tag and just follow that and remove rest of the existing tags/releases.

Reply:
ok. I'll take a look.